nss-day-cohort-39/tabloid-full-stack-vanjies
# Update post

### What I changed:
1. added edit button to posts posted by current user that open a modal form for user to updated his/her posts.

### How to test:
1. check that any post by the current logged in user has an edit button... whether in myposts, posts or post details view.
2. check that the button, when clicked, opens a modal form that is populated with the post's data.
3. check that you can change the default data
4. upon clicking save button, the post should be updated and relisted in the posts/myposts views or the postdetails view (depending on where you started from)
